src/test/java/org/folio/rest/api/CancellationReasonsApiTest.java

```diff
@@ -256,7 +258,7 @@ public void canDeleteCancellationRequest()
     assertCreateCancellationReason(request);
     assertDeleteCancellationReason(id);
     JsonResponse getResponse = getCancellationReason(id);
-    assertTrue(getResponse.getStatusCode() == 404);
+    assertEquals(HttpStatus.HTTP_NOT_FOUND.toInt(), getResponse.getStatusCode());
```

Comment:
I'm starting to think that a custom matcher that expects a FOLIO `HttpStatus` and the actual value is either the status code or the response itself, might be useful in improving the readability of these assertions.

Reply:
Thanks for introducing some custom matchers to make this more readable